infrastructure-scripts/claude_start.sh

```
@@ -125,23 +121,55 @@ start_orchestration_background() {
     fi
 
     # Start orchestration silently in background
+    # NOTE: start_system.sh starts long-running services, so we don't use timeout on it
     (
         cd "$SCRIPT_DIR/orchestration"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)